qt/widgets/common/src/PythonHelpWindow.cpp

```diff
@@ -0,0 +1,90 @@
+#include "MantidQtWidgets/Common/PythonHelpWindow.h"
+#include "MantidQtWidgets/Common/PythonHelpBridge.h"
+#include <QString>
+#include <string>
+
+namespace MantidQt {
+namespace MantidWidgets {
+
+#ifndef DOCS_QTHELP
```

Comment:
You missed turning off `REGISTER_HELPWINDOW(HelpWindow)` in the `HelpWindow.cpp` based on the other case of this macro